File: antigravity/brain/182174db-080b-469e-82bf-3f6de2c73a36/scratch/extract_transcript.py
import json

transcript_path = r"C:\Users\66830\.gemini\antigravity\brain\c65c70e4-1473-4c30-bb91-f17f7cbd18d7\.system_generated\logs\transcript.jsonl"

with open(transcript_path, 'r', encoding='utf-8') as f:
    for line in f:
        try:
            data = json.loads(line)
            if data.get('type') == 'USER_INPUT':
                print(f"Step {data.get('step_index')}: {data.get('content')}")
                print("-" * 50)
        except Exception as e:
            pass

File: antigravity/brain/182174db-080b-469e-82bf-3f6de2c73a36/scratch/extract_transcript.js
const fs = require('fs');
const readline = require('readline');

const transcriptPath = 'C:\\Users\\66830\\.gemini\\antigravity\\brain\\c65c70e4-1473-4c30-bb91-f17f7cbd18d7\\.system_generated\\logs\\transcript.jsonl';

const fileStream = fs.createReadStream(transcriptPath);
const rl = readline.createInterface({
  input: fileStream,
  crlfDelay: Infinity
});

rl.on('line', (line) => {
  try {
    const data = JSON.parse(line);
    if (data.type === 'USER_INPUT') {
      console.log(`Step ${data.step_index}: ${data.content}`);
      console.log("-".repeat(50));
    }
  } catch (e) {
    // Ignore parse errors
  }
});
